assets/js/blocks/product-query/variations/product-query.tsx

```diff
@@ -40,7 +40,7 @@ if ( isExperimentalBuild() ) {
 			[
 				'core/post-template',
 				{},
-				[ [ 'core/post-title' ], [ 'core/post-featured-image' ] ],
+				[ [ 'core/post-title' ], [ 'woocommerce/product-image' ] ],
```

Comment:
For the product grid, the image is usually to put above the title, so I think it makes sense to have it default here.
```suggestion
				[ [ 'woocommerce/product-image' ], [ 'core/post-title' ] ],
```